src/QMCWaveFunctions/TWFGrads.hpp

```diff
@@ -24,15 +24,19 @@ struct TWFGrads;
 template<>
 struct TWFGrads<CoordsType::POS>
 {
-  void resize(const std::size_t size);
+  TWFGrads(const std::size_t size) : grads_positions(size){};
```

Comment:
I saw a few places with function definition moved to cpp files. Thumb up.
Let us leave the constructor implementation in the cpp as well.

Reply:
moved constructor to cpp